build.gradle.kts

```diff
@@ -25,16 +25,13 @@ repositories {
 }
 
 dependencies {
-    implementation(platform("org.scijava:pom-scijava:31.1.0"))
-    annotationProcessor("org.scijava:scijava-common:2.88.1")
+    implementation(platform("org.scijava:pom-scijava:35.1.1"))
+    annotationProcessor("org.scijava:scijava-common:2.94.1")
```

Comment:
You could consider defining any hardcoded versions as `val`s, e.g.: 
```kotlin
val pom_scijava_version = "35.1.1"
implementation(platform("org.scijava:pom-scijava:$pom_scijava_version"))
```
Since it's much more common to bump dep versions than it is to change dependency groupId/artifactId.

But I don't have a strong opinion.

Reply:
I'd rather have it together with the GAV for consistency 👍